packages/csv-parser-price/src/cli.js

```diff
@@ -85,14 +87,12 @@ Convert commercetools price CSV data to JSON.`,
   })
 
   .option('logLevel', {
-    default: 'info',
     describe: 'Logging level: error, warn, info or verbose.',
   })
-  .coerce('logLevel', (arg) => {
-    npmlog.level = arg
-  })
   .argv
 
+npmlog.level = args.logLevel || minLogLevel
```

Comment:
I still don't get how this solves the issue

Let's have a call about this later